permits/templates/permits/permit_request_detail.html

```diff
@@ -133,11 +133,11 @@ <h2>{% trans "Données du service pilote" %}</h2>
         </div>
       </div>
     {% endif %}
-    
+
     {% if validations %}
     <div class="row sub-row">
       <div class="col">
-        <h2>{% trans "Commentaires des services consultés" %}</h2>
+        <h2>{% trans "Validation" %}</h2>
```

Comment:
Sure ?

Reply:
or Validation**s** ?

Reply:
> Renommer "Commentaires des services consultés" --> "Validation" (car les commentaires sont facultatifs, le requérant peut ne voir que la pastille de validation)

Validation seems more global, and there can be only one validation...